File: web_edi_api-development/src/main/java/jp/co/jun/edi/util/BooleanUtils.java
package jp.co.jun.edi.util;

/**
 * Booleanユーティリティ.
 */
public final class BooleanUtils {

    /**
     */
    private BooleanUtils() {
    }

    /**
     * 値が1の場合、true を返す.
     *
     * @param value 値
     * @return boolean型
     */
    public static boolean toBoolean(final String value) {
        if (value == null) {
            return false;
        }

        return "1".equals(value);
    }
}

File: web_edi-master/src/app/util/boolean-utils.ts
/**
 * 真偽型ユーティリティ.
 */
export class BooleanUtils {
  constructor(
  ) { }

  /**
   * string型のboolean値をboolean型へ変換します。
   * 'true'でも'false'でもない場合はnullを返します。
   *
   * @param str 文字列型の真偽値
   * @returns boolean型に変換した値
   */
  static parseStrToBoolean(str: string): boolean {
    switch (str) {
      case 'true':
        return true;
      case 'false':
        return false;
      default:
        return null;
    }
  }
}
